### PR TITLE
Add --print_kernel_info

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

File: clients/include/testing_matmul.hpp [hunk 1]
```diff
@@ -2026,15 +2029,60 @@ void testing_matmul(const Arguments& arg)
...
+        if(arg.print_kernel_info)
+        {
+            solutionIndex = hipblaslt_ext::getIndexFromAlgo(heuristicResult[best_sol].algo);
+            solutionName
+                = hipblaslt_ext::getSolutionNameFromAlgo(handle, heuristicResult[best_sol].algo);
+            kernelName
+                = hipblaslt_ext::getKernelNameFromAlgo(handle, heuristicResult[best_sol].algo);
+        }
+
+        hipblaslt_cout << "Winner: " << std::endl;
```
Comment (jichangjichang) [inline clients/include/testing_matmul.hpp:2074]: only need when user request multiple solutions

Comment (KKyang) [reply clients/include/testing_matmul.hpp]: fixed

**Review by jichangjichang — APPROVED** [review]
LGTM

**Review by hcman2 — APPROVED** [review]
Good for ticket SWDEV-425876

**Review by Serge45 — APPROVED** [review]

**Pull request merged**